### PR TITLE
Add Hacktoberfest 2019 - MayaData

### LINKED ISSUE #318: Hacktoberfest 2019: MayaData (OpenEBS & Litmus)
## Wild swag opportunity appeared!

- [x] I've checked that this isn't a duplicate issue.

Q. Which company offers it?  
A. : MayaData

Q. Where have you seen the announcement (URL)?  
A. : https://twitter.com/mayadata_inc/status/1177725829138305029

Q. What kind of swag (clothing, stickers, ...)?  
A. : TBD

Q. Have you received swag from this?  
A. : no

Q. How long did it take?  
A. : N/A

Q. Does it ship worldwide?  
A. : Dunno

- [x] I've checked that the promotion is legit.
- [x] I've checked that it's still available to the public.

### PR DESCRIPTION
**Pull request by GuyKh**
Add MyData.io Hacktoberfest offer

- [x] I've checked that this isn't a new swag opportunity proposal.
- [x] I've checked that this isn't a duplicate pull request.

 
Closes #318

### CODE REVIEW

**Comment by aslafy-z** [comment]
It is mayadata.io not mydata.io. Can you change it?

**Comment by hugovk** [comment]
It is MayaData not mayadaya.io :)

File: data.json [hunk 1]
```diff
@@ -179,6 +179,15 @@
     "dateAdded": "2018-10-23T10:17:39.000Z",
     "tags": ["clothing", "hacktoberfest", "expired"]
   },
+  {
+    "name": "MayaData.io",
```
Comment (hugovk) [inline data.json]: MayaData not MayaData.io :)

Comment (GuyKh) [reply data.json]: I think I fixed that, could you check again?

Comment (hugovk) [reply data.json]: 👍

File: data.json [hunk 2]
```diff
@@ -179,6 +179,15 @@
     "dateAdded": "2018-10-23T10:17:39.000Z",
     "tags": ["clothing", "hacktoberfest", "expired"]
   },
+  {
+    "name": "MayaData",
+    "difficulty": "medium",
+    "description": "At the end of the month-long event, the individual who makes the most significant contributions will receive a brand new laptop.",
+    "reference": "https://blog.mayadata.io/mayadata-celebrates-hacktoberfest-2019",
+    "image": "https://blog.mayadata.io/hs-fs/hubfs/Hacktoberfest-11-1.png",
+    "dateAdded": "2018-03-11T16:47:14.000Z",
```
Comment (vikaspotluri123) [inline data.json]:
```suggestion
    "dateAdded": "2018-09-30T16:47:00.000Z",
```

This date should be a lot closer to today

File: data.json [hunk 3]
```diff
@@ -179,6 +179,15 @@
     "dateAdded": "2018-10-23T10:17:39.000Z",
     "tags": ["clothing", "hacktoberfest", "expired"]
   },
+  {
+    "name": "MayaData",
+    "difficulty": "medium",
+    "description": "At the end of the month-long event, the individual who makes the most significant contributions will receive a brand new laptop.",
```
Comment (vikaspotluri123) [inline data.json]: It looks like you can also get other swag for contributing in general

Comment (GuyKh) [reply data.json]: 👍

File: data.json (hunk 2 above)
Comment (GuyKh) [reply data.json]: 👍

File: data.json [hunk 4]
```diff
@@ -179,6 +179,15 @@
...
     "tags": ["clothing", "hacktoberfest", "expired"]
   },
+  {
+    "name": "MayaData",
+    "difficulty": "medium",
+    "description": "Get swag for contributing in general. At the end of the month-long event, the individual who makes the most significant contributions will receive a brand new laptop.",
+    "reference": "https://blog.mayadata.io/mayadata-celebrates-hacktoberfest-2019",
+    "image": "https://blog.mayadata.io/hs-fs/hubfs/Hacktoberfest-11-1.png",
+    "dateAdded": "2019-10-02T16:48:14.000Z",
+    "tags": ["clothing", "hacktoberfest", "stickers", "laptop"]
```
Comment (aslafy-z) [inline data.json]: Please change `laptop` tag to `device`

Comment (aslafy-z) [reply data.json]:
```suggestion
    "tags": ["clothing", "hacktoberfest", "stickers", "device"]
```

**Pull request merged**